frontend/pulse3d/src/pages/upload-form.js

```diff
@@ -193,36 +211,26 @@ export default function UploadForm() {
         endTime,
         selectedPulse3dVersion,
       } = analysisParams;
-
-      const requestBody = {
-        upload_id: uploadId,
-        baseline_widths_to_use: formatTupleParams(baseToPeak, peakToBase),
-        prominence_factors: formatTupleParams(
-          prominenceFactorPeaks,
-          prominenceFactorValleys
-        ),
-        width_factors: formatTupleParams(widthFactorPeaks, widthFactorValleys),
-        twitch_widths: twitchWidths === "" ? null : twitchWidths,
-        start_time: startTime === "" ? null : startTime,
-        end_time: endTime === "" ? null : endTime,
-        // pulse3d versions are currently sorted in desc order, so pick the first (latest) version as the default
-        version:
-          selectedPulse3dVersion === ""
-            ? pulse3dVersions[0]
-            : selectedPulse3dVersion,
-      };
-      if (requestBody.version !== "0.24.6") {
-        // Tanner (9/15/21): at the time of writing this, 0.24.6 is the only available pulse3D version that does not support the max_y param
-        requestBody.max_y = maxY === "" ? null : maxY;
-      }
-
-      const jobResponse = await fetch(
-        `${process.env.NEXT_PUBLIC_PULSE3D_URL}/jobs`,
-        {
-          method: "POST",
-          body: JSON.stringify(requestBody),
-        }
-      );
+      const jobResponse = await fetch("https://curibio.com/jobs", {
```

Comment:
There are definitely some merge issues here, but if you also made changes to this request you'll need to work those in as well